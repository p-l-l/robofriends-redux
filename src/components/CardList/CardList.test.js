import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

describe('CardList component', () => {

    it('Renders correctly', () => {
        const mockRobots = [
            {
                id: 1,
                name: 'John Snow',
                username: 'JohnJohn',
                email: 'john@gmail.com'
            }
        ]
        expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
    });

});