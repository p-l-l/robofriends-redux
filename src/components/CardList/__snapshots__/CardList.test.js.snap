// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardList component Renders correctly 1`] = `
<div>
  <Card
    email="john@gmail.com"
    id={1}
    key="0"
    name="John Snow"
  />
</div>
`;
